test/libyul/yulOptimizerTests/equalStoreEliminator/mcopy_same_write_area_mcopy.yul

```diff
@@ -0,0 +1,23 @@
+{
+    let _0 := 0
+    let _32 := 32
+    let _64 := 64
+
+    calldatacopy(0, 0, 0x60)
+
+    mcopy(_0, _32, _32)
+    mcopy(_0, _64, _32) // Not redundant. Copying from a different spot.
```

Comment:
I mean, I wouldn't call this `Not redundant.`, just not the same; if this were `UnusedStoreEliminator`, the first copy would be removed (if we had that implemented).

Reply:
But it's not redundant. The comment is about the second instruction.

Well, maybe I should comment on the fist one that it is redundant, but in these tests the focus is on the second one really.